src/plugins/gs-plugin-eos.c

```diff
@@ -173,7 +173,7 @@ gs_plugin_initialize (GsPlugin *plugin)
 
 	/* let the flatpak plugins run first so we deal with the apps
 	 * in a more complete/refined state */
-	gs_plugin_add_rule (plugin, GS_PLUGIN_RULE_RUN_AFTER, "flatpak");
+	gs_plugin_add_rule (plugin, GS_PLUGIN_RULE_RUN_AFTER, "flatpak-system");
```

Comment:
Wow! I thought I had changed this when Richard renamed the system plugin...
Thanks! I have just tested (because it could have implications) but everything looks fine.